drivers/dac/dac_mcux_lpdac.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+ * Copyright (c) 2023, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define DT_DRV_COMPAT nxp_lpdac
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/dac.h>
+#include <zephyr/logging/log.h>
+
+#include <fsl_dac.h>
+
+LOG_MODULE_REGISTER(dac_mcux_lpdac, CONFIG_DAC_LOG_LEVEL);
+
+struct mcux_lpdac_config {
+	LPDAC_Type *base;
+	dac_reference_voltage_source_t refVoltage;
```

Comment:
Please use snake_case for variable names:

```suggestion
	dac_reference_voltage_source_t ref_voltage;
```